tests/test_layout_objects.py

```diff
@@ -440,16 +440,17 @@ def test_tab_and_tab_holder(self):
                     "one",
                     "first_name",
                     css_id="custom-name",
-                    css_class="first-tab-class active",
+                    css_class="first-tab-class",
```

Comment:
```suggestion
                    css_class="first-tab-class active",
```
I've add this fix because we don't have to add `active` css class manually. Tests should to check is the class added automatically.